ntp/syncTime.go

```diff
@@ -105,6 +128,13 @@ func (s *syncTime) sync() {
 
 			if err != nil {
 				log.Error(fmt.Sprintf("NTP Error: %s", err))
+				//Change host if current host return an error
```

Comment:
I would replace/remove now the constant totalRequests with len(s.ntpOptions.Host) in line 126. We might have maybe more than 10 ntp servers added in config and we should give a chance to anyone of them not only to the first 10(totalRequests).

Reply:
Done.